Dockerfile

```
@@ -20,7 +21,7 @@ SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 # Minify Assets
 # hadolint ignore=DL3008
 RUN apt-get update \
-  && apt-get install -y --no-install-recommends minify \
+  && apt-get install -y --no-install-recommends minify libwebp-dev \
```

Maybe newline for better diff readability 
```suggestion
  && apt-get install -y --no-install-recommends \
  minify \
  libwebp-dev \
```